### PR TITLE
Added adaptive timing and headers disable controls

### PR DESCRIPTION
**Pull request by Pho3niX90**

### CODE REVIEW

File: src/main/java/pt/lighthouselabs/obd/commands/protocol/AdaptiveTimingObdCommand.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	}
+	
+	@Override
+	public String getFormattedResult() {
+		return getResult();
+	}
+
+	@Override
+	public String getName() {
+		return "Echo Off";
```
Comment (pires) [inline src/main/java/pt/lighthouselabs/obd/commands/protocol/AdaptiveTimingObdCommand.java]: This is not true.

File: src/main/java/pt/lighthouselabs/obd/commands/protocol/HeadersOffObdCommand.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	}
+	
+	@Override
+	public String getFormattedResult() {
+		return getResult();
+	}
+
+	@Override
+	public String getName() {
+		return "Echo Off";
```
Comment (pires) [inline src/main/java/pt/lighthouselabs/obd/commands/protocol/HeadersOffObdCommand.java]: This is not true.

**Comment by pires** [comment]
Indentation: 2 spaces.

`mvn clean package` will take care of adding the license and running the tests. Can you add tests, please?

**Comment by Pho3niX90** [comment]
Sorry I am not familiar with that command?

**Comment by Pho3niX90** [comment]
I fixed the issues in the files. Will be more careful next time

**Comment by pires** [comment]
@Pho3niX90 how are you building this library? Officially, it's built with [Maven](https://maven.apache.org/), hence `mvn clean package` builds the JAR.

**Comment by anti43** [comment]
if you open the whole project in netbeans, it runs the maven commands if
you click on run, test, clean etc

2015-07-22 20:26 GMT+02:00 Paulo Pires <notifications@github.com>:

**Comment by Pho3niX90** [comment]
@pires Well basically I downloaded the repository and imported it into android studio. The maven dependancies was automatically imported.

I will have a look at maven with android studio.

**Pull request merged**